### PR TITLE
Create like button, done button and alert - feature UNO-116

### PR DESCRIPTION
**Pull request by ErciliaR**
This PR should cover:
- like button UI (missing request)
- done button UI 
- alert UI (missing request)

PS.: I add an ExperienceDetailsView to see how it would work together but this file will change over the development of this view as a whole.

Those components would require UI Test, but I'm not able to do it. If someone could help me out, I can add it here. To do UI test, you just have to change the view activated by the Experiences tab to ExperienceDetailsView or go to preview mode over components.

### CODE REVIEW

**Review by Bassani-Paulo — APPROVED** [review]

**Review by Fercomp — APPROVED** [review]

File: WhyNot/WhyNot/Components/DoneButton/DoneButton.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+import SwiftUI
+
+struct DoneButton: View {
+    
+    @State private var showingConfirmAlert: Bool = false
+    @Binding var isExperienceDone: Bool
+    @Binding var isIconEnabled: Bool
+    @Binding var experienceTitle: String
+    let buttonHeight = 44.0
```
Comment (LuisNascimento11) [inline WhyNot/WhyNot/Components/DoneButton/DoneButton.swift:16]: transformar isso num view model

File: WhyNot/WhyNot/Scenes/Experiences/ExperiencesDetailsView.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+//  Created by Ercília Regina Silva Dantas on 14/11/22.
+//
+
+import SwiftUI
+
+struct ExperiencesDetailsView: View {
+    @State private var experienceTitle: String = "Jump jump"
+    @State private var isExperienceDone: Bool = false
+    @State private var isIconEnabled: Bool = true
+    @State private var isLiked: Bool = true
```
Comment (LuisNascimento11) [inline WhyNot/WhyNot/Scenes/Experiences/ExperiencesDetailsView.swift:14]: transformar num view model

File: WhyNot/WhyNot/Scenes/Experiences/ExperiencesDetailsView.swift [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    let safetySpace = 36.0
+    let iconBtnSize = 22.0
+    let categoryIconWidth = 224.0
```
Comment (LuisNascimento11) [inline WhyNot/WhyNot/Scenes/Experiences/ExperiencesDetailsView.swift:17]: enum com as constantes

Comment (luizgustavo09) [reply WhyNot/WhyNot/Scenes/Experiences/ExperiencesDetailsView.swift:17]: Concordo

**Review by luizgustavo09 — APPROVED** [review]
LGTM!

File: WhyNot/WhyNot/Components/DoneButton/DoneButton.swift (hunk 1 above)
Comment (ErciliaR) [reply WhyNot/WhyNot/Components/DoneButton/DoneButton.swift:16]: fixed

File: WhyNot/WhyNot/Scenes/Experiences/ExperiencesDetailsView.swift (hunk 3 above)
Comment (ErciliaR) [reply WhyNot/WhyNot/Scenes/Experiences/ExperiencesDetailsView.swift:17]: done

**Pull request closed without merging**